react-app/src/components/Common/newbutton.js

```diff
@@ -53,7 +53,7 @@ const NewButton = () => {
 
   const onClinkFinish = values => {
     setIsLoading(true);
-    dispatch(addClink(values.clinkTitle, currentToken, addSuccess, addFail))
+    dispatch(addClink(values.clinkTitle, values.privacy, currentToken, addSuccess, addFail))
```

Comment:
Set it to be false if it doesn't exist
e.g `dispatch(addClink(values.clinkTitle, values.privacy || false, currentToken, addSuccess, addFail))`